asserts/asserts_test.go

```diff
@@ -778,7 +778,7 @@ func (as *assertsSuite) TestWithAuthority(c *C) {
 		"validation",
 		"repair",
 	}
-	c.Check(withAuthority, HasLen, asserts.NumAssertionType-3) // excluding device-session-request, serial-request, account-key-request
+	c.Check(withAuthority, HasLen, asserts.NumAssertionType-4) // excluding device-session-request, serial-request, account-key-request
```

Comment:
Actually, I should have added the new type to the `withAuthority` list. Fixed now.